Formula/p/python-importlib-resources.rb

```diff
@@ -0,0 +1,30 @@
+class PythonImportlibResources < Formula
```

Comment:
I dont think this needs to be a formula since its only needed for older pythons (its [builtin to python3.8+](https://docs.python.org/3/library/importlib.metadata.html))

arcade-learning-environment doesn't need it, you can patch in https://github.com/Farama-Foundation/Arcade-Learning-Environment/pull/491 if you want instead

Reply:
Done in #138182